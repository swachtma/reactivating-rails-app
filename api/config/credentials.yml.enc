NruSqAfDHjCumPIyJn6xsY9LOZuE3ZzX2o/1QTdVkugVslYlIp4h/gmrnNPKDUKYZ8vGEGHPJ5EwlOSClaonGjgUB4yUG4tz8I0T++XLbiEU8qhxATlTf+s9JNfusXdgIB2ZFO4ABSDKhxhzPLbHYJVvLghx3WmwMMpO+GBgzlp94DLCX2IR38jeHEydFPy73oteodKVbTBp0g91v8DhpnZUTWHYb0JdqS+otrdrS1mar9MlcMnTpb8chvLisrrvE0SR/QUagAbd5UD/d02NczFYmAdKsyztbryVyakhgosZhCJFY3KNwxayxDC3sF5xFQHgli8qJNXNAhzaYDVf+n7D9k6g9BHTV5YRXdICgYORVWo+a9EhjBTmVwQsJN+IrdZaqxQVoa2DMEA3CVjOlp1wtLNwHbHArmId--8+ArDlMZvB3FEYTs--m4O75Mk62PJos0rR8zw91Q==