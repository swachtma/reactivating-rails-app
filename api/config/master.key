dc4eeb306d00d2a5e6993b94e449787e