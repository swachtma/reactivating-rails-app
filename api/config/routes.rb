Rails.application.routes.draw do
  scope :api do
    get 'github/', to: "authentication#github", format: false
    get 'hydrate_user/', to: "authentication#show"
    get 'nodes/', to: "nodes#index"
    get 'chapters/', to: "chapters#index"
    resources :bookmarks, only: [:index, :create]
  end
end