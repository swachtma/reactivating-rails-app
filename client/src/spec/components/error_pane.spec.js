/* global expect, jest */
import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';

import config from '../../test_helper';
import ErrorPane from '../../components/error_pane';

let w = shallow(<ErrorPane />);

describe("<ErrorPane /> renders",()=>{
  it("according to snapshot", () => expect(w).toMatchSnapshot());
});