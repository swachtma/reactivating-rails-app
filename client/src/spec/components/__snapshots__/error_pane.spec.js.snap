// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorPane /> renders according to snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ErrorPane />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <Container
        text={true}
>
        <h1>
                Well Shucks...
        </h1>
        <p>
                This page is where we, unfortunately, make up some cutesy reason to apologize for our site not working at the moment. In any case, we don't seem able to do the things at the moment.
        </p>
        <p>
                How about you 
                <Connect(Link)
                        to={
                                Object {
                                        "type": "HOME_ROUTE",
                                      }
                        }
                >
                        head to the homepage
                </Connect(Link)>
                 and try giving it another go?
        </p>
</Container>,
      "id": "error-pane",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h1>
            Well Shucks...
</h1>,
          <p>
            This page is where we, unfortunately, make up some cutesy reason to apologize for our site not working at the moment. In any case, we don't seem able to do the things at the moment.
</p>,
          <p>
            How about you 
            <Connect(Link)
                        to={
                                    Object {
                                                "type": "HOME_ROUTE",
                                              }
                        }
            >
                        head to the homepage
            </Connect(Link)>
             and try giving it another go?
</p>,
        ],
        "text": true,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Well Shucks...",
          },
          "ref": null,
          "rendered": "Well Shucks...",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "This page is where we, unfortunately, make up some cutesy reason to apologize for our site not working at the moment. In any case, we don't seem able to do the things at the moment.",
          },
          "ref": null,
          "rendered": "This page is where we, unfortunately, make up some cutesy reason to apologize for our site not working at the moment. In any case, we don't seem able to do the things at the moment.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              "How about you ",
              <Connect(Link)
                to={
                                Object {
                                                "type": "HOME_ROUTE",
                                              }
                }
>
                head to the homepage
</Connect(Link)>,
              " and try giving it another go?",
            ],
          },
          "ref": null,
          "rendered": Array [
            "How about you ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {
                "children": "head to the homepage",
                "to": Object {
                  "type": "HOME_ROUTE",
                },
              },
              "ref": null,
              "rendered": "head to the homepage",
              "type": [Function],
            },
            " and try giving it another go?",
          ],
          "type": "p",
        },
      ],
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <Container
          text={true}
>
          <h1>
                    Well Shucks...
          </h1>
          <p>
                    This page is where we, unfortunately, make up some cutesy reason to apologize for our site not working at the moment. In any case, we don't seem able to do the things at the moment.
          </p>
          <p>
                    How about you 
                    <Connect(Link)
                              to={
                                        Object {
                                                  "type": "HOME_ROUTE",
                                                }
                              }
                    >
                              head to the homepage
                    </Connect(Link)>
                     and try giving it another go?
          </p>
</Container>,
        "id": "error-pane",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": Array [
            <h1>
              Well Shucks...
</h1>,
            <p>
              This page is where we, unfortunately, make up some cutesy reason to apologize for our site not working at the moment. In any case, we don't seem able to do the things at the moment.
</p>,
            <p>
              How about you 
              <Connect(Link)
                            to={
                                          Object {
                                                        "type": "HOME_ROUTE",
                                                      }
                            }
              >
                            head to the homepage
              </Connect(Link)>
               and try giving it another go?
</p>,
          ],
          "text": true,
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Well Shucks...",
            },
            "ref": null,
            "rendered": "Well Shucks...",
            "type": "h1",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "This page is where we, unfortunately, make up some cutesy reason to apologize for our site not working at the moment. In any case, we don't seem able to do the things at the moment.",
            },
            "ref": null,
            "rendered": "This page is where we, unfortunately, make up some cutesy reason to apologize for our site not working at the moment. In any case, we don't seem able to do the things at the moment.",
            "type": "p",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                "How about you ",
                <Connect(Link)
                  to={
                                    Object {
                                                      "type": "HOME_ROUTE",
                                                    }
                  }
>
                  head to the homepage
</Connect(Link)>,
                " and try giving it another go?",
              ],
            },
            "ref": null,
            "rendered": Array [
              "How about you ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "class",
                "props": Object {
                  "children": "head to the homepage",
                  "to": Object {
                    "type": "HOME_ROUTE",
                  },
                },
                "ref": null,
                "rendered": "head to the homepage",
                "type": [Function],
              },
              " and try giving it another go?",
            ],
            "type": "p",
          },
        ],
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
